### PR TITLE
Dynamic sprites

### PR DESCRIPTION
**Pull request by Jlawlzz**
Created functionality to load custom environment sprites by level to allow for grassy/desert/whatever you would like themes. Pattern can be expanded for whatever level specific sprites desired to be added.

### CODE REVIEW

File: lib/engine.js [hunk 1]
```diff
@@ -25,9 +25,13 @@ let towerButtons = $('.tower-button');
...
 let moneyAmount = document.getElementById('money-amount');
-
 let imageLoader = new ImageLoader();
-let images = imageLoader.init();
+let images = imageLoader.init(game.currentLevel.stage);
+
+function loadImages() {
+  imageLoader = new ImageLoader();
+  images = imageLoader.init(game.currentLevel.stage);
+}
```
Comment (ToniRib) [inline lib/engine.js]: Is there a reason why these two lines are both in a function and outside of the function?

**Comment by Jlawlzz** [comment]
the loadImages function is for when we need to reload images at the start of a level.  It doesnt like that I instantiate the images variable inside the function so I figured id do it once outside of the function upon start of the game.  I can push up an alternative where I set images to null before the function so the variable is set on a file wide scope. Let me know which way you prefer.

**Comment by ToniRib** [comment]
So you can actually just say `let images` all by itself outside of the function to put it in the correct scope but not set it to anything. JS is weird like that.

**Comment by ToniRib** [comment]
Anyway, it's not a big deal and I don't care that much one way or another.

**Comment by Jlawlzz** [comment]
Oh cool just saw this, ill do let images and push back up

**Comment by Jlawlzz** [comment]
Made the change, should be ready for merge

**Pull request merged**